### DISCUSSION TITLE
Send Alert from Tradingview

### DISCUSSION THREAD

**Discussion by llerch** [FabCx/TradingView-Webhook-Bot#59] [Q&A]
![image](https://user-images.githubusercontent.com/45877247/121410983-706bf280-c963-11eb-8950-d26f7d01345c.png)

How can I send an Alert like in the Screenshot. For me it's only possible to send Alerts in the JSON format with msg and these attributes.

**Comment by fabston** [comment]
Hi there,

I am not sure what you mean. You would have to set the `msg` tag in the json string.

**Reply by llerch** [reply]
Okey, thanks for your fast reply. I thought it would be possible without it because of the screenshot. btw i like your bot :)

**Reply by fabston** [reply]
The screenshot runs on an older version of the bot which did not support json. But the result is the same.

**Reply by Audiamant** [reply]
Hello i installed the bot and i would ask if it's possible to send the msg in in different lines like in the screenshot you posted 

my msg looks like this now 
![image](https://user-images.githubusercontent.com/33065755/180651135-a6874e10-b918-4b4a-87a3-c1fa5227394a.jpeg)👇🏽👇🏽

**Reply by fabston** [reply]
Hi there. 

If you’d need a line break use `/n`. Hope that helps.

**Reply by Audiamant** [reply]
thank you now it works like I want it to .


\n

**Reply by Audiamant** [reply]
I have another question regarding the alert message. Is it possible to save the text in the message box e.g. for future alerts?

If not is there another way?
maybe with a script ?

**Reply by fabston** [reply]
It’s possible but for that it might be the best to use a database then. Such as MySQL, Postgres etc.
